payjoin/src/core/receive/optional_parameters.rs

```diff
@@ -36,6 +36,35 @@ impl Default for Params {
 }
 
 impl Params {
+    /// when only one parameter is present rather than failing the entire payjoin process.
+    ///
+    /// This  allows for graceful degradation and doesn't halt the payjoin process
+    /// due to incomplete optional parameters, while still alerting about the suboptimal
+    /// configuration that prevents fee adjustment capability.
+    ///
+    /// # Arguments
+    /// * `max_additional_fee_contribution` - Optional maximum fee contribution amount
+    /// * `additional_fee_output_index` - Optional index of output to adjust for fees
+    /// * `
+    fn handle_additonal_fee_param(
+        max_additional_fee_contribution: Option<bitcoin::Amount>,
+        additional_fee_output_index: Option<usize>,
+        params: &mut Params,
```

Comment:
Could you not pass mutable reference to *just* the `params.additional_fee_contribution` rather than the whole Params struct?